vos/vos/commands/vchmod.py

```diff
@@ -190,20 +190,23 @@ def vchmod():
                         vospace_token=opt.token,
                         insecure=opt.insecure)
         node = client.get_node(opt.node)
-        if opt.recursive:
-            node.props.clear()
-            node.clear_properties()
-            # del node.node.findall(vos.Node.PROPERTIES)[0:]
+        node.props.clear()
+        node.clear_properties()
         if 'readgroup' in props:
             node.chrgrp(props['readgroup'])
         if 'writegroup' in props:
             node.chwgrp(props['writegroup'])
         if 'ispublic' in props:
             node.set_public(props['ispublic'])
         logging.debug("Node: {0}".format(node))
-        status = client.update(node, opt.recursive)
-        if status:
-            sys.exit(status)
+        successes, failures = client.update(node, opt.recursive)
+        if opt.recursive:
```

Comment:
Is it ok that users are only notified of success/failure if the operation is recursive? Is this information useful in a 
 `vchmod *py` scenario?

Reply:
True. I've made all the recursive operations quiet by default. To receive a report, one would need to run them at least in verbose mode. It is more consistent with the OS commands.
BTW, wild cards are no supported in `vchmod` and a lot of other commands. Users have been asking for it and at one point we were considering supporting them on the server side. That looks less likely now, so we'll probably need to do something on the client side.